### DISCUSSION TITLE
Wayland alternative to xephyr and xpra?

### DISCUSSION THREAD

**Discussion by ghost** [netblue30/firejail#6521] [Q&A]
In x11, firejail can use xephyr or xpra to open an isolated desktop, how to open an isolated desktop on Wayland?

**Comment by rusty-snake** [comment]
If you want an "isolated *desktop*" you can start an nested session with your preferred compositor (weston, mutter, ...).

Otherwise Wayland does not suffer from the trusted mis-design in X11. Support for security context is tracked in #5883.
